awx/ui/client/features/inventory-scripts/inventory-scripts.strings.js

```diff
@@ -0,0 +1,28 @@
+function InventoryScriptsStrings (BaseString) {
+    BaseString.call(this, 'applications');
+
+    const { t } = this;
+    const ns = this.applications;
+
+    ns.state = {
+        TITLE: t.s('INVENTORY SCRIPTS'),
+        LIST_BREADCRUMB_LABEL: t.s('INVENTORY SCRIPTS'),
+        ADD_BREADCRUMB_LABEL: t.s('CREATE INVENTORY SCRIPT'),
+        EDIT_BREADCRUMB_LABEL: t.s('EDIT INVENTORY SCRIPT')
+    };
+
+    ns.list = {
+        ROW_ITEM_LABEL_ORGANIZATION: t.s('ORG'),
+        ROW_ITEM_LABEL_MODIFIED: t.s('LAST MODIFIED'),
+        DELETE_TOOLTIP: t.s('Delete Inventory Script'),
+        COPY_TOOLTIP: t.s('Copy Inventory Script')
```

Comment:
once my i18n PR lands, we could probably grab COPY and DELETE from the `base-string.service.js`
https://github.com/ansible/tower/pull/2736/files#diff-b9d14a6ae22dbe8a695e07950ce966abR96

Reply:
(and once AWX gets the release branch code)